docs_src/src/pages/documentation/api_reference/scaling.mdx

```diff
@@ -0,0 +1,29 @@
+## Scaling the Application
+
+Batman scaled his application across multiple cores for better performance. He used the following command:
+
+<Row>
+  <Col>
+  </Col>
+  <Col sticky>
+    <CodeGroup title="Request">
+
+      ```python {{ title: 'untyped' }}
+      python3 app.py --workers=N --process=M
+      ```
+
+      ```python {{title: 'typed'}}
+      python3 app.py --workers=N --process=M
+      ```
+    </CodeGroup>
+  </Col>
+</Row>
+
+---
+
+## What's next?
+
+Now, Batman wanted to extend Robyn. Robyn told him about the plugins.
+
+- [Plugins](/documentation/plugins)
```

Comment:
could probably make another issue for this but now that I look at it, the navigation does not match the layout, we should probably talk about the plugins at the end.

Reply:
moved to #844